src/ui/default-prop-types.js

```diff
@@ -509,3 +510,17 @@ export const favoriteRoomShape = PropTypes.shape({
 export const favoriteDocumentShape = PropTypes.shape({
   ...contributedDocumentMetadataProps
 });
+
+// This fits both EVENT_TYPE.revisionCreated and EVENT_TYPE.commentCreated,
+// but we may need different params later
+const documentEventParamsShape = PropTypes.shape({
```

Comment:
Maybe we could rename it to `documentEvent[Common/Base]ParamsShape` and drop the comment?

Reply:
Dropped!